### PR TITLE
preflight 0.8.1

### PR DESCRIPTION
**Pull request by wanda-carlson**
preflight 0.8.1

### CODE REVIEW

**Comment by wanda-carlson** [comment]
**Henry Richard** you need to login to be tagged!

**Comment by henry-richard** [comment]
@wanda-carlson this is pretty cool

File: src/wanda.py [hunk 1]
```diff
@@ -3,3 +3,4 @@
 print("Wanda testing v0.8.0") # local
 print("Wanda testing v0.8.0 Preflight 0.1")
 print("Wanda testing notifications 0.2")
```
Comment (henry-richard) [inline src/wanda.py:5]: is it necessary to keep this?

File: src/wanda.py [hunk 2]
```diff
@@ -3,3 +3,4 @@
...
+print("Wanda testing preflight 0.8.1")
```
Comment (henry-richard) [inline src/wanda.py:6]: pre flight is one or two words?

**Review by henry-richard — CHANGES_REQUESTED** [review]
minor improvements

File: src/wanda.py (hunk 2 above)
Comment (wanda-carlson) [reply src/wanda.py:6]: preflight is one word

File: src/wanda.py (hunk 1 above)
Comment (wanda-carlson) [reply src/wanda.py:5]: yes

**Review by henry-richard — APPROVED** [review]

**Pull request merged**